jobs/build_debian/build_debian.groovy

```diff
@@ -0,0 +1,49 @@
+node(build_debian_node){
+    lock("debian"){
+        timestamps{
+            withEnv([
+                "MANIFEST_FILE_URL=${env.MANIFEST_FILE_URL}",
+                "IS_OFFICIAL_RELEASE=${env.IS_OFFICIAL_RELEASE}",
+                "BINTRAY_SUBJECT=rackhd-mirror",
+                "CI_BINTRAY_REPO=debian",
+                "BINTRAY_COMPONENT=main",
+                "BINTRAY_DISTRIBUTION=trusty", 
+                "BINTRAY_ARCHITECTURE=amd64"]) {
+
+                deleteDir();
+                def shareMethod
+                dir("Build_Debian_JFiles"){
+                    checkout scm
+                    shareMethod = load("jobs/shareMethod.groovy")
+                }
+                def url = "https://github.com/RackHD/on-build-config.git"
+                def branch = "*/master"
+                def targetDir = "on-build-config"
+                shareMethod.checkout(url, branch, targetDir)
+
+                withCredentials([
+                    usernameColonPassword(credentialsId: "ff7ab8d2-e678-41ef-a46b-dd0e780030e1", 
```

Comment:
Nice. 
the credentialsId is binding to Jenkins server. (aka, it's not quite portable if community run this groovy in their Jenkins. so at least add comments in code. )